kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-common/src/main/java/org/kie/server/remote/rest/common/util/RestUtils.java

```diff
@@ -94,12 +97,23 @@ public static Response createResponse(Object responseObj, Variant v, javax.ws.rs
         return responseBuilder.build();
     }
 
-    
-    public static Variant getVariant(HttpHeaders headers) { 
+    public static Variant getVariant(HttpHeaders headers) {
+        return getVariant(headers, null);
+    }
+
+    public static Variant getVariant(HttpHeaders headers, Object obj) {
         Variant v = RestEasy960Util.getVariant(headers);
         if( v == null ) {
             v = Variant.mediaTypes(getMediaType(headers)).add().build().get(0);
         }
+
+        if (obj instanceof ItemList || (obj instanceof ServiceResponse && ((ServiceResponse) obj).getResult() instanceof ItemList)) {
```

Comment:
@mswiderski I calculated the objects being ItemList as the only ones aggregatable. I don't see if others must be in that regard.